root/Makefile

```diff
@@ -96,9 +96,14 @@ pre-commit: fmt
 .PHONY: build
 build:: pre-build gobuild
 
+## lint:            run code linters
+.PHONY: lint
+lint:: pre-test
```

Comment:
should you just make a new hook, `pre-lint`?

Reply:
Prolly, for compat I'll run pre-test tho